test/e2e/validation_test.go

```diff
@@ -433,39 +433,48 @@ func (tc *testContext) getWinServices(addr string) (map[string]winService, error
 func testExpectedServicesRunning(t *testing.T) {
 	tc, err := NewTestContext()
 	require.NoError(t, err)
-
-	ownedByCCM, err := cluster.IsCloudControllerOwnedByCCM(tc.client.Config)
+	expectedSvcs, err := tc.expectedWindowsServices(windows.RequiredServices)
 	require.NoError(t, err)
-
 	for _, node := range gc.allNodes() {
 		t.Run(node.GetName(), func(t *testing.T) {
 			addr, err := controllers.GetAddress(node.Status.Addresses)
 			require.NoError(t, err, "unable to get node address")
 			svcs, err := tc.getWinServices(addr)
 			require.NoError(t, err, "error getting service map")
-			for _, svcName := range windows.RequiredServices {
+			for svcName, shouldBeRunning := range expectedSvcs {
```

Comment:
Is the Azure Cloud Manager service always present on the instance even on non-Azure clouds? Is it that it will just not be running?

Reply:
That service will only run on Azure instances